.github/bin/check-nightly-olm-bundle.sh

```diff
@@ -71,21 +71,29 @@ installOperatorSDK() {
 }
 
 isActualNightlyOlmBundleCSVFiles() {
-  cd "${ROOT_PROJECT_DIR}"
+  local CSV_FILE_KUBERNETES="deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml"
+  local CSV_FILE_OPENSHIFT="deploy/olm-catalog/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml"
+  local CRD_FILE_KUBERNETES="deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/org_v1_che_crd.yaml"
+  local CRD_FILE_OPENSHIFT="deploy/olm-catalog/eclipse-che-preview-openshift/manifests/org_v1_che_crd.yaml"
+
   export NO_DATE_UPDATE="true"
   export NO_INCREMENT="true"
-  source "${ROOT_PROJECT_DIR}/olm/update-nightly-bundle.sh"
 
-  CSV_FILE_KUBERNETES="deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml"
-  CSV_FILE_OPENSHIFT="deploy/olm-catalog/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml"
+  cd "${ROOT_PROJECT_DIR}"
+  source "${ROOT_PROJECT_DIR}/olm/update-nightly-bundle.sh"
```

Comment:
should we call `update-crd-files.sh` as well ?

Reply:
We should not because it is already called here: https://github.com/eclipse/che-operator/blob/master/.github/bin/check-nightly-olm-bundle.sh#L32